### PR TITLE
chore(modules): remove refactor module

### PR DESCRIPTION
**Pull request by steelsojka**

### CODE REVIEW

**Review by vigoux — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -320,6 +246,13 @@ require'nvim-treesitter.configs'.setup {
 EOF
 ```
 
+# External Modules
```
Comment (stsewd) [inline README.md:249]: I have wanted to add this section since a long time!

```suggestion
# External modules
```

**Review by stsewd — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -320,6 +246,13 @@ require'nvim-treesitter.configs'.setup {
...
+
+Other modules can be installed as plugins.
+
+- [refactor](https://github.com/nvim-treesitter/refactor) - Refactoring and definition modules
+- [playground](https://github.com/nvim-treesitter/playground) - Treesitter integrated playground
```
Comment (theHamsta) [inline README.md:254]: You could add your documentation plugin?

**Review by theHamsta — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (stsewd) [reply README.md:254]: I think we can just link to the wiki now :)

Comment (steelsojka) [reply README.md:254]: I'm gonna wait to add that one. It's still super raw and is going to change a bit once we get language injection.

**Pull request merged**